### PR TITLE
Migrate to GitHub actions

### PR DESCRIPTION
**Pull request by davidbruce**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+      - uses: krdlab/setup-haxe@v1
```
Comment (NobbZ) [inline .github/workflows/test.yml:10]: Do you really need haxe available when running `configlet`?

**Review by NobbZ — COMMENTED** [review]
Besides the comment I did inline, it seems as if you aren't running any haxe specific tests, not even trying to compile files?

File: .github/workflows/test.yml (hunk 1 above)
Comment (davidbruce) [reply .github/workflows/test.yml:10]: I am fine with removing the setup-haxe.  I'll add it back if I get some test generation setup.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+        with:
+          haxe-version: 4.0.5
```
Comment (ErikSchierboom) [inline .github/workflows/test.yml:11]: I think this is a remnant of a previous iteration.

```suggestion
        with:
          haxe-version: 4.0.5
```

**Review by ErikSchierboom — APPROVED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (davidbruce) [reply .github/workflows/test.yml:11]: Whoops, absolutely right.

**Review by ErikSchierboom — APPROVED** [review]
LGTM

**Pull request merged**